tests/components/yamaha/test_media_player.py

```diff
@@ -86,13 +86,17 @@ async def test_setup_host(hass: HomeAssistant, device, device2, main_zone) -> No
     assert state.state == "off"
 
 
-async def test_setup_attribute_error(hass: HomeAssistant, device, main_zone) -> None:
-    """Test set up integration encountering an Attribute Error."""
+async def test_setup_find_errors(hass: HomeAssistant, device, main_zone) -> None:
```

Comment:
Rather parametrize this test to better simulate the real life

Reply:
not quite sure what you where after but have had a go at something better?